cli/README.md

```diff
@@ -82,6 +82,7 @@ path|status|
 
 path|status|description
 -|-|-
+[jobs/dataprep/dask/dask-job.yml](jobs/dataprep/dask/dask-job.yml)|[![jobs/dataprep/dask/dask-job](https://github.com/Azure/azureml-examples/workflows/cli-jobs-dataprep-dask-dask-job/badge.svg)](https://github.com/Azure/azureml-examples/actions/workflows/cli-jobs-dataprep-dask-dask-job.yml)|*no description*
```

Comment:
consider adding description in the README to get picked up here

Reply:
picked up from the job YAML itself, not the README*

Reply:
not sure what you mean -- can you elaborate?